notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -18,6 +18,59 @@ https://en.wikipedia.org/wiki/List_of_science_fiction_films_of_the_2000s
+joins forces with the surviving crew and other passengers to develop a plan to
+escape the planet.
+
+:: Date
+2000-02-18
+
+:: References
+https://en.wikipedia.org/wiki/Pitch_Black_(film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+what if I were stranded on a deserted planet [The story follows the efforts of the surviving crew and passengers of a crash-landed spaceship to get off a desert planet that was inhabited by hoards of man-eating creatures.]
+desert planet [A main novelty of the film is the baked in perpetual daylight by three suns desert planet on which it is set.]
+human vs. nature [The surviving crew and passengers of the Hunter-Gratzner spaceship struggled to survive on the desert planet on which they crash-landed.]
+trust in a potential foe [Carolyn and Johns took a calculated risk by releasing the dangerous criminal Riddick and accepting him into their party. They remained suspicious and mistrustful of him throughout the course of the film, but needed his help to survive on the planet.]
+human vs. monster [Central to the story was the surviving crew and passengers of the Hunter-Gratzner spaceship trying to not get killed by the man-eating creatures that inhabited the desert planet on which they'd crash-landed.]
+monster [The desert planet was inhabited by grotesque, man-eating creatures.]
+human character stereotype [The hero of the story Riddick was a stereotypical hard-boiled, lone wolf, in-it-for-only-himself Hollywood antihero figure.]
+character metamorphosis [The story pointedly concluded with Riddick stated that "he died somewhere on that planet", meaning that he'd left his old life as an infamous criminal and murderer behind there, and was now a reformed man.]
+speculative ability [A main novelty of the story was that Riddick had surgically altered eyes that allowed him to see in the dark.]
+
+:: Minor Themes
+speculative spaceship [The futuristic interstellar vessel Hunter-Gratzner was transporting 40 passengers in cryostasis.]
+cryogenic technology [The Hunter-Gratzner spaceship was transporting 40 cryo-sleeping passengers.]
+habitable planet in a trinary star system [Three suns featured prominently in the desert planet sky.]
+organized religion [The Muslim preacher Imam and his three young acolytes were traveling to a place called New Mecca.]
+role models [The tomboy Jack immediately took a liking to the hardened criminal Riddick and soon thereafter began to emulate him.]
+fear [The marooned Hunter-Gratzner crew and passengers pointedly discussed their fears, or lack thereof, of the vicious creatures that went flying around the desert planet at night, killing anything in their path.]
+gender bending [The young girl Jack was posing as a boy to avoid attention.]
+personal practical dilemma [Riddick forced Carolyn to choose between leaving her fellow marooned travelers for dead on the planet and escaping with him, and staying behind on the planet to die together with them.] 
```

Comment:
I compare with https://www.themeontology.org/theme.php?name=sacrificing%20a%20friend%20to%20save%20one%27s%20self and think we might do well to introduce "abandoning someone to save one's self"..

Reply:
I added a stub entry. Will define in another pull request.